xml/admin_install_salt.xml

```diff
@@ -1159,10 +1171,11 @@ master_minion: <replaceable>MASTER_MINION_NAME</replaceable>
      </listitem>
      <listitem>
       <para>
-       <emphasis>ganesha</emphasis> - the minion will act as an &ganesha;
+       <emphasis>nfs-ganesha</emphasis> - the minion will act as an &ganesha;
```

Comment:
```suggestion
       <emphasis>ganesha</emphasis> - the minion will act as an &ganesha;
```
my `ls /srv/pillar/ceph/proposals/` states `role-ganesha`, not `role-nfs-ganesha`